### ISSUE TITLE
Why do not use default list style?

### ISSUE THREAD

**Issue by Aqua-Dream** [theme-next/hexo-theme-next#722]
### I agree and want to create new issue 

- [X] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [X] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [X] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
Note that in `post-expand.styl` the style of list is set to be `list-style: circle`. But why do not use the default behavior, which is more popular?

 - Default Behavior
   - List-Style: circle

**Comment by 1v9** [comment]
Every style has its option in `_config.yml` is impossible and crazy, so there are many styles are hardcoded for now.
If you want new [list-style](https://developer.mozilla.org/zh-CN/docs/Web/CSS/list-style-type), set a custom styl, for example:
```css
.posts-expand .post-body {
  ul > li {
    list-style: square;
  }
}
```

**Comment by Aqua-Dream** [comment]
I understand you, and I know how to set the style in custom file. I just doubt that setting "list-style: circle" may be a historical reason (https://github.com/iissnan/hexo-theme-next/commit/3b3f31bd8eb3eba5ff2f05fdeba310adc3aa5c79), which may be not necessary now, so I suggest using the default behavior. I've never seen a markdown render works like this way.

**Comment by 1v9** [comment]
Yes, no need anymore, circle is strange.

**Comment by stevenjoezhang** [comment]
I agree, there are actually a lot of redundant styles...

**Comment by ivan-nginx** [comment]
```
- sdfdsf
- sdfdsf
  - sdfdsf
  - sdfsdf

* sdfdsf
* sdfdsf
  * sdfdsf
  * sdfsdf
```

## Current state

![image](https://user-images.githubusercontent.com/16944225/54828187-f254a000-4cb3-11e9-8273-1fb3c7d0c9bd.png)

## Without `list-style: circle`

![image](https://user-images.githubusercontent.com/16944225/54828250-0e584180-4cb4-11e9-9072-a3e9ae14fc3e.png)

**Comment by stevenjoezhang** [comment]
Fixed in #728.

If you have any recommendations or find bugs, feel free to open an new issue.
